docs/feature_ps2_mouse.md

```diff
@@ -88,6 +88,32 @@ In your keyboard config.h:
 #endif
 ```
 
+### Interrupt Version (ARM chibios)
+
+pretty much any two pins can be used for the (software) interrupt variant on ARM cores, the example below uses A8 for clock, and A9 for data.
+
+NOTE: due to implementation details of the EXT config, "any two pins", is currently limited to the Port-A (see tmk_core/protocol/ps2_interrupt.c)
+
+In rules.mk:
+
+```
+PS2_MOUSE_ENABLE = yes
+PS2_USE_INT = yes
+```
+
+In your keyboard config.h:
+
+```
+#define PS2_LINE_CLOCK PAL_LINE(GPIOB, 8U)
+#define PS2_LINE_DATA  PAL_LINE(GPIOA, 9U)
```

Comment:
This should be aligned with other config options
```suggestion
#define PS2_LINE_DATA B9
```

Reply:
corrected with f6369f72a